dataset.proto

```diff
@@ -34,9 +40,11 @@ message DataSet {
     string url = 1;
   }
 
-  Categorization categorization = 1;
-  AdFraud        adfraud        = 2;
-  // 3 and 4 are reserved
-  Malicious      malicious      = 5;
-  Echo           echo           = 6;
+  Categorization         categorization = 1;
+  AdFraud                adfraud        = 2;
+  // 3 is reserved
+  // if SENTIMENT is not requested, keywords will map to zero values  
+  map<string, Sentiment> keyword        = 4;
```

Comment:
Yes, Sentiment and Keyword should be kept separate at this level, just as they are separate from Categorization or Echo.  Any mappings are calculated relational information above these base atoms.

This is a good time to reiterate keeping our Pivotal stories small and pointed.  With separate stories for Keyword vs Sentiment messages, inadvertently "crossing the streams" like this would have been much less likely.